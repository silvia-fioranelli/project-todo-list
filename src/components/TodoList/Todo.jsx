import { shape, string } from 'prop-types';

const Todo = (props) => {
  const { item } = props;
  return <div>{item.content}</div>;
};

Todo.propTypes = {
  item: shape({
    content: string.isRequired,
  }),
};

export default Todo;
