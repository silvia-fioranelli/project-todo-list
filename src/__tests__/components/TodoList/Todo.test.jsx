import { shallow } from 'enzyme';
import Todo from '../../../components/TodoList/Todo';

describe('Todo', () => {
  describe('rendering', () => {
    test('should render item', () => {
      const props = {
        item: { content: 'mock content', id: 'mock id' },
      };
      const wrapper = shallow(<Todo {...props} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
