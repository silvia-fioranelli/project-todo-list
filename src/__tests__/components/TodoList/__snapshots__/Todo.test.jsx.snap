// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo rendering should render item 1`] = `
<div>
  mock content
</div>
`;
